air.toml

```diff
@@ -0,0 +1,6 @@
+[format]
+line-width = 120
```

Comment:
I set it at 120, the same as the current r-polars, because I feel that 80 is too short in this day and age, but 120 may be too long.
https://github.com/pola-rs/r-polars/blob/d0e1d6a31d7f0c624e52ee316c861e6595e4ee60/.lintr.R#L2

@etiennebacher Thoughts?

Reply:
I think 80 (or close to 80) is recommended or the default in several other places:

* [tidyverse style guide](https://style.tidyverse.org/syntax.html#long-function-calls)
* Black recommends [88 characters](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length)
* I think Ruff also enforces 80 by default but I can't find a link

120 is definitely too long for me, especially since I'm working on laptop these days. I'd be willing to try 90 or 100 max.

Reply:
Ok, the Linux kernel has relaxed the limit from 80 to 100. 100 looks good.